src/main/cpp/hw/ctreadapters/pro/DragonPositionInchToCTREProAdapter.cpp

```diff
@@ -27,21 +28,43 @@
 #include "utils/ConversionUtils.h"
 
 // Third Party Includes
+#include "units/length.h"
+#include "ctre/phoenixpro/controls/PositionDutyCycle.hpp"
+#include "ctre/phoenixpro/controls/PositionTorqueCurrentFOC.hpp"
+#include "ctre/phoenixpro/controls/PositionVoltage.hpp"
 
+using ctre::phoenixpro::controls::PositionDutyCycle;
+using ctre::phoenixpro::controls::PositionTorqueCurrentFOC;
+using ctre::phoenixpro::controls::PositionVoltage;
 using ctre::phoenixpro::hardware::TalonFX;
 using std::string;
 
 DragonPositionInchToCTREProAdapter::DragonPositionInchToCTREProAdapter(string networkTableName,
                                                                        int controllerSlot,
                                                                        const ControlData &controlInfo,
                                                                        const DistanceAngleCalcStruc &calcStruc,
-                                                                       DragonTalonFX &controller) : DragonControlToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
+                                                                       ctre::phoenixpro::hardware::TalonFX &controller) : DragonControlToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
 {
 }
 
 void DragonPositionInchToCTREProAdapter::Set(double value)
 {
-    // TODO  Add phoenix pro commands
+    units::angle::degree_t target = units::angle::turn_t(value / (m_calcStruc.diameter * std::numbers::pi));
```

Comment:
Why aren't we using the countsPerInch value from calcStruc here?  If we keep using diameter, how would we find the values for something like our extender last year?

Reply:
Updated; thanks